core/pom.xml

```diff
@@ -22,6 +22,7 @@
         <netbeans.hint.license>apache20</netbeans.hint.license>        
         <maven.build.timestamp.format>yyyy-MM-dd'T'HH:mm:ssZ</maven.build.timestamp.format>        
         <builddate>${maven.build.timestamp}</builddate>        
+        <geotools.version>13.2</geotools.version>
```

Comment:
You will want to update this to the latest maintainance release (14.x). The current stable (15.x requires java 8)